plugin/evm/limit_order.go

```diff
@@ -367,13 +378,46 @@ func (lop *limitOrderProcesser) loadMemoryDBSnapshot() (acceptedBlockNumber uint
 	var snapshot orderbook.Snapshot
 	err = gob.NewDecoder(buf).Decode(&snapshot)
 	if err != nil {
-		return acceptedBlockNumber, fmt.Errorf("Error in snapshot parsing; err=%v", err)
+		return acceptedBlockNumber, fmt.Errorf("Error in snapshot parsing from hubbleDB; err=%v", err)
 	}
 
 	if snapshot.AcceptedBlockNumber != nil && snapshot.AcceptedBlockNumber.Uint64() > 0 {
 		err = lop.memoryDb.LoadFromSnapshot(snapshot)
 		if err != nil {
-			return acceptedBlockNumber, fmt.Errorf("Error in loading from snapshot: err=%v", err)
+			return acceptedBlockNumber, fmt.Errorf("Error in loading snapshot from hubbleDB: err=%v", err)
+		} else {
+			log.Info("ListenAndProcessTransactions - memory DB snapshot loaded from hubbleDB", "acceptedBlockNumber", acceptedBlockNumber)
+		}
+
+		return snapshot.AcceptedBlockNumber.Uint64(), nil
+	} else {
+		return acceptedBlockNumber, nil
+	}
+}
+
+func (lop *limitOrderProcesser) loadMemoryDBSnapshotFromFile() (acceptedBlockNumber uint64, err error) {
```

Comment:
remvoed loadMemoryDBSnapshotFromHubbleDB now

Reply:
added the function back, but it will be removed soon

Reply:
yes but looks like we can abstract out most of the common logic in another function

Reply:
since `loadMemoryDBSnapshotFromHubbleDB` will be removed anyway, i feel it can be kept as is. The different functions also have method specific logs which I want to keep

Reply:
[clarification] once we deploy these changes to prod and start using the snapshot file, the hubble-db code will essentially be dead. we are adding a task to remove that code which includes `loadMemorySnapshotFromHubbledb`, so there will be no duplication left in the code. 

hope this helps, let me know if you have other questions or were talking about something entirely else